index.html

```diff
@@ -283,5 +283,30 @@ <h2>Skills</h2>
       <img clas="arrowimg" src="normarrow.png">
     </div>
   </section>
+  <section id="footer">
+    <footer class="flex">
+      <h2>Contact me</h2>
+      <p>If you have an application you are interested in developing, a feature that you need built or a 
+      project that needs coding. I’d love to help with it 
+      </p>
+      <form action="https://formspree.io/f/{form_id}" method="post">
```

Comment:
- [ ] I kindly suggest that you create an account on formspree.io, so that you can get your unique form id that you can use for your `https://formspree.io/f/{form_id}`;
 - [ ] Please ensure that your form submits successfully.
 
 The below is the message I get when I try to submit a form.
<img width="453" alt="image" src="https://user-images.githubusercontent.com/52934128/192627185-43bc7b90-19ff-410d-96e3-5d8b2bc1d4e7.png">